PurchasesCoreSwift/FoundationExtensions/Error+Extensions.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Error+Extensions.swift
+//
+//  Created by Joshua Liebowitz on 8/6/21.
+
+import Foundation
+
+@objc public extension NSError {
+
+    var rc_successfullySynced: Bool {
+        if code == ErrorCode.networkError.rawValue {
+            return false
+        }
+
+        if userInfo[Backend.RCSuccessfullySyncedKey as String] == nil {
+            return false
+        }
```

Comment:
seems like this block is unnecessary, right? i.e.: if the key is not there, it'll already `return false` from line 27

Reply:
Yup!